packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/checkpoints/checkpointService.test.ts

```diff
@@ -682,31 +683,31 @@ describe('Checkpoint parsing from SourceBreakpoint', () => {
 });
 
 describe('Verify SFDX Toggle Checkpoint callback, sfdxToggleCheckpoint', () => {
-  const cpService = require('../../../src/breakpoints/checkpointService');
-
   const breakpointEnabled = true;
   const uriInput = vscode.Uri.parse('file:///bar.cls');
   const lineInput = 5;
 
-  // These need to be stubbed in order to not require an open file in an active editor with a selection.
-  // tslint had to be disabled for these two variables because, being stubs, they're not directly called
-  // in here and it'll cause lint to fail.
-  /* tslint:disable */
-  const fetchActiveEditorUriStub = sinon
-    .stub(cpService, 'fetchActiveEditorUri')
-    .returns(uriInput);
-  const fetchActiveSelectionLineNumberStub = sinon
-    .stub(cpService, 'fetchActiveSelectionLineNumber')
-    .returns(lineInput - 1);
-  /* tslint:enable */
```

Comment:
I saw no good reason for this to be outside a beforeEach. Moved to a more standard setup for tests.